os/net/mac/framer/framer-802154.c

```diff
@@ -92,6 +94,10 @@ create_frame(int do_create)
   framer_802154_setup_params(packetbuf_attr, packetbuf_holds_broadcast(),
                              &params);
 
+  params.dest_pid = 1 == broadcastpan
```

Comment:
shouldn't this be done in `framer_802154_setup_params` instead?

Reply:
yeah makes more sense.